packages/dd-trace/src/datastreams/processor.js

```diff
@@ -13,16 +13,14 @@ const log = require('../log')
 
 const ENTRY_PARENT_HASH = Buffer.from('0000000000000000', 'hex')
 
-const HIGH_ACCURACY_DISTRIBUTION = 0.0075
-
 class StatsPoint {
   constructor (hash, parentHash, edgeTags) {
     this.hash = hash.readBigUInt64BE()
     this.parentHash = parentHash.readBigUInt64BE()
     this.edgeTags = edgeTags
-    this.edgeLatency = new LogCollapsingLowestDenseDDSketch(HIGH_ACCURACY_DISTRIBUTION)
```

Comment:
Although this is equivalent behavior, I do have some concerns that this is not the correct behavior

@ida613 (author) what is the downside of lower accuracy for sketches?

Reply:
I believe the difference does not matter much for now, since we never received complains about this.

Reply:
Sounds good - it can definitely be changed after the fact, so resolving my comment